graylog2-web-interface/src/pages/InputsPage.tsx

```diff
@@ -36,7 +41,20 @@ const InputsPage = () => {
     <DocumentTitle title="Inputs">
       <div>
         <PageHeader title="Inputs">
-          <span>Graylog nodes accept data via inputs. Launch or terminate as many inputs as you want here.</span>
+          {isCloud
+            ? (
+              <>
+                <p> Graylog cloud accepts data via inputs. There are many types of inputs to choose from, but only some
+                  can run directly in the cloud. You can launch and terminate them on this page.
+                </p>
+                <p>
+                  {/* eslint-disable-next-line react/no-unescaped-entities */}
+                  If you are missing an input type on this page's list of available inputs, you can start the input on
```

Comment:
```suggestion
                  If you are missing an input type on this page&apos;s list of available inputs, you can start the input on
```

Reply:
This should work as well and we can get rid of the eslint-disable. I am usually trying to avoid those if possible.